frontend/e2e/helpers.cafe.ts

```diff
@@ -295,6 +295,10 @@ export const createFeature = async (
   description = 'description',
 ) => {
   await gotoFeatures()
+  await t.eval(() => {
```

Comment:
Hmm, what's the theory behind this solving issues with e2e ?

Reply:
That function will scroll down the screen. One issue is that I was trying to click on a component that wasn't visible.